svc-managers/managers/managers.go

```diff
@@ -440,6 +445,15 @@ func fillResponse(body []byte, managerData map[string]interface{}) response.RPC
 		return common.GeneralError(http.StatusInternalServerError, response.InternalError, err.Error(),
 			[]interface{}{}, nil)
 	}
+	if _, ok := respData["DateTime"]; !ok {
```

Comment:
This should not be checked as manager for odimra and plugin should always populate current datetime

Reply:
addressed